x-pack/legacy/plugins/maps/public/layers/sources/es_geo_grid_source/geo_tile_utils.js

```diff
@@ -37,15 +37,30 @@ function sinh(x) {
   return (Math.exp(x) - Math.exp(-x)) / 2;
 }
 
+// Calculate the minimum precision required to adequtely draw the box
```

Comment:
I think we should add a comment that the `DECIMAL_DEGREES_PRECISION` value (set to 5, and I doubt will ever change) will be used for zooms < 21.  For above 21, it will use 6+, a few zooms per prescision.  @nreese do we allow such high zooms?

Reply:
@nyurik 

The screenshots in #44303 that demonstrate this problem were produced using the map at zoom levels supported out-of-the box (i.e. the problem fixed in the PR was based off the allowable zoom level supported by the map).

For what it's worth, the code here does not deal with the zoom level of the map, but rather the zoom level that is passed into the `geotile_grid` aggregation.   With the most-fine grid setting the aggregation is performed at four zoom levels lower than the current map zoom level.

https://github.com/elastic/kibana/blob/01daabcb53224203764bafd28acc6c76dd3af6f9/x-pack/legacy/plugins/maps/public/layers/sources/es_geo_grid_source/es_geo_grid_source.js#L155

I believe the max zoom of the map itself is 24.

https://github.com/elastic/kibana/blob/367da246985e311687b5c3af08c1538a9e9a26cc/x-pack/legacy/plugins/maps/public/connected_components/layer_panel/layer_settings/layer_settings.js#L27

So that means the `geotile_grid` zoom level will be up to 28.